### DISCUSSION TITLE
Linking Cart Items back to product page - On cart page

### DISCUSSION THREAD

**Discussion by amandeepsinghvirdi** [bigcommerce/catalyst#718] [General]
**Detail:**
On the cart page, the cart items are not linked back to the product page. Please see the video for more explanation:
https://somup.com/cZeUDTCOF8

In cornerstone theme - the item title is linked back to product page. But I think it will be good to link "Product Image" and "Product Title" both linking back to product page. Better UX.

**Comment by jorgemoya** [comment]
Hi @amandeepsinghvirdi!

Thank you for the feedback. We'll take a look and consider it for a future release!

**Reply by amandeepsinghvirdi** [reply]
I have added this in my project already. Should I go ahead and create PR for this?

**Reply by jorgemoya** [reply]
Sure! There's some considerations that need to be done but open to your suggestions!
